### DISCUSSION TITLE
AfterUpdate ReducerProtocol getting error from other feature

### DISCUSSION THREAD

**Discussion by saroar** [pointfreeco/swift-composable-architecture#1472] [General]
<img width="1437" alt="Screenshot 2022-10-12 at 10 47 25" src="https://user-images.githubusercontent.com/8770772/195282490-53ec13e6-0b9f-43cd-acd8-a000bdfe3863.png">

hi after update TCA I am select and working on one feature but getting error from other feature which before I dont remember I had this problem is it only I am getting error other other too 
thanks

**Comment by mbrandonw** [comment]
Can you pin TCA to 0.40.0 and confirm it compiles there?

As far I can tell in the screenshot, there is nothing in 0.41.0 that would cause this problem. It seems Xcode cannot find the `WordEnvironment` type. Perhaps its in another module, or maybe there are compiler errors in that type?
